.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,48 @@
+name: Feature request
+description: Suggest a new feature
+labels: enhancement
+
+body:
+  - type: markdown
+    id: description
+    attributes:
+      value: |
+          ## Before posting a feature request
+          Search existing GitHub issues to make sure the issue does not already exist:
+          https://github.com/xanaduai/pennylane/issues
+
+          For general technical details check out our documentation:
+          https://pennylane.readthedocs.io
+
+          # Feature description
+          Description of the feature request - include code snippets and screenshots here if relevant.
+  - type: textarea
+    id: details
+    attributes:
+      label: Feature details
+      description: |
+        What feature would you like to have?
+    validations:
+      required: true
+  - type: textarea
+    id: implementation
+    attributes:
+      label: Implementation
+      description: |
+        Do you have an idea for how this could be implemented? Please include those details here.
+  - type: dropdown
+    id: urgency
+    attributes:
+      label: How urgently would you need this feature?
```

Comment:
Just wondering if instead of urgency, highlighting importance would be slightly more descriptive :thinking: It might be a feature would be great to have asap, though it's not entirely too important :smile:

Reply:
So, rather "How important to you think this feature is?" Would that be better to have?